src/test/common/services/tokenApi.test.js

```diff
@@ -51,6 +54,19 @@ describe('UserSessions', () => {
       });
   });
 
+  it('setMfaToken rejects', () => {
```

Comment:
These particular reject tests might need applying to a lot of the service tests, to increase code coverage.